netbox_netprod_importer/push.py

```diff
@@ -141,6 +144,20 @@ def _push_interfaces(self):
                 self._handle_interface_mode(interface, if_prop["mode"])
                 if_prop.pop("mode")
 
+            if if_prop["untagged_vlan"]:
+                vlan_id = self._get_vlan_id(if_prop["untagged_vlan"])
+                if vlan_id != -1:
+                    setattr(interface, "untagged_vlan", vlan_id)
+
+            if_prop.pop("untagged_vlan")
+            if len(if_prop["tagged_vlans"]):
+                setattr(interface, "tagged_vlans", [])
+                for vlan in if_prop["tagged_vlans"]:
+                    vlan_id = self._get_vlan_id(vlan)
+                    if vlan_id != -1:
+                        interface.tagged_vlans.append(vlan_id)
+            if_prop.pop("tagged_vlans")
```

Comment:
Same comment as before, you'll get a `KeyError` if `tagged_vlans` is not defined.

Reply:
"tagged_vlan" is always there, is being formed: importer.py:177